crates/cxx-qt-gen/src/parser/cxxqtdata.rs

```diff
@@ -43,6 +50,8 @@ impl ParsedCxxQtData {
         Self {
             cxx_mappings: ParsedCxxMappings::default(),
             qobjects: BTreeMap::<Ident, ParsedQObject>::default(),
+            qenums: Vec::default(),
```

Comment:
why `Vec::default()` vs just `vec![]` like elsewhere?

Reply:
same for two lines below

Reply:
No reason, really... can change.